.github/workflows/integration-tests.yml

```diff
@@ -16,28 +16,24 @@ env:
   IMAGE_AGENT: ${{ github.repository_owner }}/rancher-agent
 jobs:
   test:
-    runs-on: runs-on,runner=8cpu-linux-x64,image=legacy-cgroups-for-x64,run-id=${{ github.run_id }}
+    runs-on:
+      - runs-on
+      - spot=capacity-optimized
+      - runner=32cpu-linux-x64
```

Comment:
That's quite a jump from 8 cores to 32. FWIW https://github.com/rancher/rancher/pull/47789 bumped it from 8 to 16 so this one is asking twice that.

I we're also changing from `spot=price-capacity-optimized` to `spot=capacity-optimized` so there is extra run cost here.

Is there an increased CPU requirement associated with cgroupsv2 enabled or marked speed up of tests execution?

Maybe we should consider 8 CPU cores + `spot=false` instead as potentially the same cost and better stability?